src/routes/brand/brandRoutes.ts

```diff
@@ -11,7 +11,7 @@ export default class BrandsRoutes {
 
     this.router.post('/', createBrandValidator, async (req, res, next) => {
       try {
-        const data = await brandsService.createBrand(req.body.name, req.body.logo, req.body.website, req.body.contact);
+        const data = await brandsService.createBrand(req.body.name, req.body.logo, req.body.website, req.body.contact, req.body.bowie_brands_id, req.body.demographics, req.body.media_handles, req.body.industry, req.body.comments, req.body.last_updated_by, req.body.last_modified_at);
```

Comment:
last modified by and last modified at should not come from req body
for lastmodifiedby you can use Auth userId and lastmodified by you can use Current timestamp